README.md

```diff
@@ -2,7 +2,6 @@ Classnames
 ===========
 
 [![NPM version](https://badgen.net/npm/v/classnames)](https://www.npmjs.com/package/classnames)
```

Comment:
```suggestion
[![NPM version](https://badgen.net/npm/v/classnames)](https://www.npmjs.com/package/classnames)
![Node.js CI](https://github.com/JedWatson/classnames/workflows/Node.js%20CI/badge.svg)
```

Here's the Github Actions status badge.
You might wonder why I added a `%20` to the URL after `Node.js`, that's to replace the spacebar character in the URL.

Reply:
Thank you @HonkingGoose!  I intend to pick up this PR in the coming week,  but these suggestions/commits have been well explained and I appreciate it.

Reply:
Thank you! 😄 ❤️